### PR TITLE
basic intro page and delete templates

### PR DESCRIPTION
**Pull request by robertswh**
wanted to get a quick intro page on as we've began to share the link.

### CODE REVIEW

File: ons-spark/intro.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
-ONS staff should also look at our GitLab pages for organisation-specific information.
+Some of our core **values** are:
+- Openness and transparency
+- Diversity and collaboration
+- Good coding practices
+- Efficiency and value for money
 
-Open to contributions from anyone, across UK Govt and beyond.
\ No newline at end of file
+This book is open to contributions from anyone across UK Govt and beyond.
```
Comment (ad-prince) [inline ons-spark/intro.md:13]: I'd spell out Government in full

File: ons-spark/intro.md [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+
+## Further resources
+
+Some examples of other free resources:
+
+- [Learning Spark 2nd edition](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf?utm_medium=email&utm_source=databricks&utm_campaign=7014N0000026tqzQAA)
+- [Spark official documentation](https://spark.apache.org/docs/latest/)
+- [PySpark documentation](https://spark.apache.org/docs/latest/api/python/)
+- [sparklyr documentation](https://spark.rstudio.com/)
+- [Spark by examples](https://sparkbyexamples.com/)
```
Comment (ad-prince) [inline ons-spark/intro.md:25]: Is this an appropriate place to link through to Arthurs book and the Duck Book?

File: ons-spark/intro.md (hunk 1 above)
Comment (robertswh) [reply ons-spark/intro.md]: done

File: ons-spark/intro.md (hunk 2 above)
Comment (robertswh) [reply ons-spark/intro.md]: done

**Pull request merged**